mappings/net/minecraft/world/chunk/ChunkDataAccess.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_5571 net/minecraft/world/chunk/ChunkDataAccess
```

Comment:
Access doesn't have any specific implications about where the data is coming from, and it might be worth clarifying that it loads from disk in the name?

Reply:
do we have a naming scheme for "loads from disk"? `Io` maybe?

Reply:
I am pretty sure that has been used in other places, yeah

Reply:
`ChunkDataIO` is good if it's not taken (do we have `ChunkIO` already?)